cron/bq/main.go

```diff
@@ -148,6 +148,22 @@ func getBQConfig() (projectID, datasetName, tableName string, err error) {
 	return
 }
 
+func getBQConfig2() (projectID, datasetName, tableName string, err error) {
```

Comment:
Simply update `getBQConfig` to return `tableName` and `tableNameV2`?

Reply:
Nvm, I just realized something. You don't need to make any code changes in this file at all. Instead, you can simply run a different instance of BQ transfer job which transfer the V2 JSON data to the V2 table.

Reply:
so basically I would create a new k8's `transferV2.yaml` and set some env variables to define the name of tables and buckets `SCORECARD_BIGQUERY_TABLE: scorecard2` and `SCORECARD_DATA_BUCKET_URL : gs://ossf-scorecard-data2`.
Is my understanding correct?

Reply:
Yep, thats correct.

Reply:
thanks. And I suppose I'll also need to create a `transfer.releaseV2.yaml`